docs/setup/import.md

```diff
@@ -19,6 +19,16 @@ or using tools like [Apache DistCp](../integrations/distcp.md#from-s3-to-lakefs)
 
 ## Zero-copy import
 
+lakeFS supports 3 ways to ingest objects from the object store without copying the data.
+They differ by the outcome, scale and ease of use:
+
+1. `lakectl ingest` - Use the CLI to create uncommitted objects in a branch. It will make sequential calls between the CLI and the server, which make scaling difficult.
+1. Import from the UI - A UI dialog to trigger an import to a designated import branch. It creates a commit from all imported objects.
+1. `lakefs import` - Use the lakeFS binary to create a commit from an S3 inventory file. Supported only with lakeFS with S3 storage adapters. Requires the lakeFS binary with DB access, hence might be hard to configure.
```

Comment:
Instead of "might be hard to configure" lets say like "Requires the most setup, but is capable of handling huge datasets."